NthKey/View/ContentView.swift

```diff
@@ -14,42 +14,59 @@ struct ContentView: View {
     @EnvironmentObject var appState: AppState
 
     var body: some View {
-        TabView {
-            
+        TabView(selection: $appState.selectedTab) {
             AddressesView()
                 .environmentObject(appState)
                 .tabItem {
                     Image(systemName: "house.fill")
                     Text("Addresses")
                 }
+                .tag(Tab.addresses)
             
             SignViewController(rootView: SignView())
                 .environmentObject(appState)
                 .tabItem {
                     Image(systemName: "lock.fill")
                     Text("Sign")
                 }
+                .tag(Tab.sign)
             
             SettingsView()
                 .tabItem {
                     Image(systemName: "gearshape.fill")
                     Text("Settings")
                 }
+                .tag(Tab.settings)
         }
     }
 }
 
 extension ContentView {
-    enum Tab {
+    enum Tab: Hashable {
         case addresses
         case sign
         case settings
     }
 }
 
+#if DEBUG
 struct ContentView_Previews: PreviewProvider {
     static var previews: some View {
-        ContentView()
-            .environmentObject(AppState())
+        var appState: AppState {
+            let result = AppState()
+            result.selectedTab = .sign
+            return result
+        }
+
+        let view = ContentView()
+            .environmentObject(appState)
+
+        return Group {
+            view
+
+            view
+                .preferredColorScheme(.dark)
```

Comment:
Why the preference for dark?

Reply:
@Sjors We should check each view in .light and.dark. For avoid situation when we have black text on the black background etc.

Reply:
@Sjors The best image as an answer
<img width="242" alt="blackAndWhite" src="https://user-images.githubusercontent.com/2025621/109071159-0e60e780-76fc-11eb-9a79-249d8c7b9aa7.png">